conf/application.conf

```
@@ -151,8 +151,8 @@ features {
   taskReopenAllowedInSeconds = 30
   allowDeleteDatasets = true
   # to enable jobs for local development, use "yarn enable-jobs" to also activate it in the database
-  jobsEnabled = false
-  voxelyticsEnabled = false
+  jobsEnabled = true
+  voxelyticsEnabled = true
```

undo me -> should also make the ci green

```suggestion
  jobsEnabled = false
  voxelyticsEnabled = false
```

---

oops